client/my-sites/post-type-list/max-pages-notice.jsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { setLayoutFocus } from 'state/ui/layout-focus/actions';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class PostTypeListMaxPagesNotice extends Component {
+	static propTypes = {
+		displayedPosts: PropTypes.number,
+		totalPosts: PropTypes.number,
+	};
+
+	componentWillMount() {
+		this.props.recordTracksEvent( 'calypso_post_type_list_max_pages_view' );
+	}
+
+	focusSiteSelector = event => {
+		event.preventDefault();
+
+		this.props.setLayoutFocus( 'sites' );
+	};
+
+	render() {
+		const { displayedPosts, totalPosts, translate } = this.props;
+
+		return (
+			<div className="post-type-list__max-pages-notice">
+				{ translate(
+					'%(displayedPosts)d of %(totalPosts)d posts shown' +
+						'{{br/}}to view more posts, {{a}}switch to a specific site{{/a}}.',
```

Comment:
This probably needs a singular and plural form, even though in English we would always use the plural form.  See https://github.com/Automattic/i18n-calypso#pluralization:

> It might seem redundant, but it is necessary for languages where a singular version may be used for counts other than 1.

Also, each line of this message should be a complete sentence, with punctuation and capital letters (where applicable).

Reply:
I got a suggestion from the i18n team here ( thanks @yoavf ) which I think is a good one:

> Showing %d post[s] out of %d

In code, it would look something like this:

```js
{ translate(
    'Showing %(displayedPosts)d post out of %(totalPosts)d.',
    'Showing %(displayedPosts)d posts out of %(totalPosts)d.',
    // ... arguments
) }
<br />
{ translate(
    'To view more posts, {{a}}switch to a specific site{{/a}}.',
    // ... arguments
) }
```